Bottom: 5114b0403ea94986b3cf3ffe409f7d82ebadee4a
Top:    63ccb4e50e73e4a66233b44f58b3b3e1ab1c17c2
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 15:55:33 -0500

Enable change hooks from Gitlab


---
diff --git a/docker/buildbot-master/buildbot/master.cfg b/docker/buildbot-master/buildbot/master.cfg
index bc7e7ccd..02056a49 100644
--- a/docker/buildbot-master/buildbot/master.cfg
+++ b/docker/buildbot-master/buildbot/master.cfg
@@ -91,7 +91,12 @@ c['www'] = {
                 ]),
             ],
         ),
+    'change_hook_dialects': {
+        'gitlab': {
+            'secret': os.getenv('GITLAB_WEBHOOK_SECRET')
+        }
     }
+}
 
 ####### PROJECT IDENTITY
